source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="chap-Renewing_certificates_{context}"]
+= Renewing certificates before they expire
+
+[role="_abstract"]
+{virt-product-fullname} certificates now follow a standard 398 day lifetime and *must* be renewed once per year.
+
+[WARNING]
+====
+Do _NOT_ let certificates expire because once expired, the environment becomes non-operational and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
```

Comment:
...the environment becomes "Non Responsive"

non-operational is a different thing

Reply:
changed non-operational to non-responsive.